package-parser/package_parser/processing/migration/_differ.py

```diff
@@ -0,0 +1,45 @@
+from abc import ABC, abstractmethod
+
+from package_parser.processing.api.model import Class, Function, Parameter, Result
+
+
+class AbstractDiffer(ABC):
+    @staticmethod
```

Comment:
Want to pass an instance of a subclass of this class around (dependency injection) to decide upon a concrete implementation. For this to work, all these methods must be instance methods rather than static methods (i.e. remove all the `staticmethod` decorators).